ui/src/components/Welcome/WelcomeSlideOne.vue

```diff
@@ -54,11 +55,24 @@
                     <li>Docker environment with Docker Compose</li>
                     <li>Network Time Protocol (NTP) installed and configured</li>
```

Comment:
Could we add the following so they know they can't do this on their phone?
```suggestion
                    <li> Access to a terminal window
                    <li>Network Time Protocol (NTP) installed and configured</li>
```

Reply:
Done.